bedrock/firefox/firefox_details.py

```diff
@@ -12,6 +13,11 @@
 class _ProductDetails(ProductDetails):
     bouncer_url = 'https://download.mozilla.org/'
 
+    # Note allizom.org is the production endpoint for SHA-1.
+    # It uses this because that's the only SHA-1 certificate
+    # we have that's usable. (SHA-1 certs can no longer be issued).
+    sha1_bouncer_url = 'https://download.mozilla.org/' if switch('disable-sha1-downloads') else 'https://download-sha1.allizom.org/'
```

Comment:
No we're doing the opposite here? Enabling the switch disables the sha-1 URL, hence why tests fail for me locally (since all switches default to on for dev).